packages/eslint-config-terra/package.json

```diff
@@ -30,6 +30,7 @@
     "lint": "eslint --ext .js,.jsx ."
   },
   "dependencies": {
+    "babel-eslint": "^10.1.0",
```

Comment:
Something we should keep an eye on, this looks like it's close to deprecation in favor of the @babel/eslint-parser

https://github.com/babel/babel-eslint#note-babel-eslint-is-now-babeleslint-parser-and-has-moved-into-the-babel-monorepo

Reply:
I'm a little on the fence about this being a hard dependency or a peer dependency, eslint itself is a peer dependency

Reply:
Should we pull in `@babel/eslint-parser` from the get go??

Reply:
That was my initial thought too, but there's a note indicating it to still be experimental until babel 8.

https://github.com/babel/babel/tree/master/eslint/babel-eslint-parser#this-project-is-still-experimental-and-will-be-released-with-babel-v8-you-can-track-our-progress-here-please-use-babel-eslint-in-the-meantime

However, you look at the linked progress issue it has a release in babel v7. So I'm not sure what the real status of the library is:

https://github.com/babel/babel/issues/10752

Reply:
@emilyrohrbough @StephenEsser i believe we're in the clear to switch to `@babel/eslint-parser`. those docs are from the master branch, but it looks like they also switched to `main` months ago

https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser

Reply:
arg, actually there is a peer dependency on eslint v7. 

https://github.com/babel/babel/blob/main/eslint/babel-eslint-parser/package.json#L30

unless anyone has objections, let's keep the dependency pointed to `babel-eslint` for now, then update to the scoped package on our eslint v7 uplift